aws-xray-recorder-sdk-sql/src/main/java/com/amazonaws/xray/sql/OracleConnectionUrlParser.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.xray.sql;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+/**
+ * Class for parsing Oracle database connection URLs and extracting useful metadata.
+ * Adapted from the OpenTelemetry JdbcConnectionUrlParser class under the Apache 2.0 license.
```

Comment:
For all the classes adapted, can you add a second copyright header with the copy from OTel?

Reply:
Sure, good callout.